containers/Ideas/IdeaProject.js

```diff
@@ -5,7 +5,15 @@ import IconSLI from 'react-native-vector-icons/SimpleLineIcons'
 import AsyncStorage from '@react-native-async-storage/async-storage'
 import { Button } from '@rneui/themed'
 import { LinearGradient } from 'expo-linear-gradient'
+<<<<<<< HEAD
```

Comment:
rebase went wrong here ;-)

Reply:
uff, sorry! and thanks for catching it..